modules/betsy/image_compress_betsy.cpp

```diff
@@ -36,11 +36,16 @@
 
 #include "bc1.glsl.gen.h"
 #include "bc6h.glsl.gen.h"
+#include "servers/display_server.h"
 
 static Mutex betsy_mutex;
 static BetsyCompressor *betsy = nullptr;
 
 void BetsyCompressor::_init() {
+	if (!DisplayServer::can_create_rendering_device()) {
```

Comment:
There is still a risk of crashing here since Betsy might still try to run even if it returns here. 

We should also add an early return if `compress_rd == nullptr` in `_compress()`. It should return an Error so that the compression can fall back to CPU compression

Reply:
done